### ISSUE TITLE
Replace stub implementations of routes for halight swaps

### ISSUE THREAD

**Issue by thomaseizinger** [comit-network/comit-rs#2544]
## Problem

Currently, 3 of 4 routes for the halight swaps have stub implementations:

https://github.com/comit-network/comit-rs/blob/ba0cfe268f327d733d86f231803b315483faa37b/cnd/src/http_api/routes/index.rs#L87-L145

This prevents us from actually creating swaps for those protocol combinations.

## Goal

Only support the **two** currently foreseen swap combinations with the halight protocol (to and from ERC20). This change will need to be accompanied with appropriate test changes.
The new test should demonstrate that on all `create swap` halight routes, we get a `201 Created` response.
The data should be handed over to the `NetworkBehaviour`.

Note that `lnd` needs to be started for these tests as cnd needs to be able to read the lnd certificate and macaroon when starting.

Note: Saving to the database is currently stubbed and doesn't need to be supported before doing this neither.

Full swap and implementation of `GET` routes are out of scope.

**Comment by thomaseizinger** [comment]
Blocked by #2542 because the NetworkBehaviour needs to support those combinations before we can call it.

**Comment by da-kami** [comment]
Noticed that we are already tackling this.

**Comment by luckysori** [comment]
This was done with #2717.
